seacrowd/sea_datasets/kde4/kde4.py

```diff
@@ -0,0 +1,307 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+A parallel corpus of KDE4 localization files. The corpus is available in 92 languages in total, with 4099 bitexts.
+"""
+import os
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@inproceedings{tiedemann2012parallel,
+  title={Parallel Data, Tools and Interfaces in OPUS},
+  author={Tiedemann, J{\"o}rg},
+  booktitle={Proceedings of the Eighth International Conference on Language Resources and Evaluation (LREC'12)},
+  pages={2214--2218},
+  year={2012}
+}
+"""
+
+_DATASETNAME = "kde4"
+
+_DESCRIPTION = """\
+A parallel corpus of KDE4 localization files. The corpus is available in 92 languages in total, with 4099 bitexts.
+"""
+
+_HOMEPAGE = "https://opus.nlpl.eu/KDE4.php"
+
+# TODO: Add languages related to this dataset
+_LANGUAGES = ["ind", "khm", "zlm", "tha", "vie"]
+
+_LICENSE = Licenses.UNKNOWN.value
+_LOCAL = False
+
+_URL_TEMPLATE = "https://opus.nlpl.eu/download.php?f=KDE4/v2/moses/{src}-{tgt}.txt.zip"
+
+_SUPPORTED_TASKS = [Tasks.MACHINE_TRANSLATION]
+_SOURCE_VERSION = "2.0.0"
+_SEACROWD_VERSION = "1.0.0"
+
+configs = {
```

Comment:
Hi @ssun32, would it be possible to use the languages' 3-letter ISO codes instead of these 2-letter ids for ease of reference?

Reply:
Also, please add a `__init__.py` file.

Reply:
@holylovenia Converted everything to iso639-3 and added __init__.py